pages/articles/[slug].vue

```diff
@@ -34,8 +34,7 @@ const formattedUpdateAt = computed(() => formatDateStringToHumanReadable(article
 const formattedCreatedAt = computed(() => formatDateStringToHumanReadable(article.value.datePublished))
 
 const route = useRoute()
-const siteUrl = useRuntimeConfig().public.site.url
-const linkToCurrentPage = computed(() => joinURL(siteUrl, route.path))
+const linkToCurrentPage = withSiteUrl(route.path)
```

Comment:
`withSiteUrl` is your friend here, should be fully reactive, not sure why I haven't documented this yet...